test/e2e/command/botkube.go

```diff
@@ -83,6 +83,10 @@ func (c *context) testBotkubeCommand(t *testing.T) {
 				"  - statefulsets\n" +
 				"  - storageclasses\n",
 		},
+		"BotKube notifier showconfig": {
+			command:  "notifier showconfig",
+			expected: "<@U023BECGF> notifier showconfig",
+		},
```

Comment:
Hey @anoopmsivadas,
We also need to verify response as well at https://github.com/infracloudio/botkube/pull/363/files#diff-bf42cad6d13b6d3c0201735d34a4d44aR106.
Let me know if you have any doubts

Reply:
Hey I accidentally included same commit in another pull request. I don't know how remove this commit from that PR

Reply:
I will create a new PR with required changes

Reply:
@anoopmsivadas We should validate the `expected` output of the command. E.g for `command list`, we are doing it here https://github.com/infracloudio/botkube/blob/aab011cfd60ed85b69380ea5eadebd8df14b64e9/test/e2e/command/botkube.go#L110. Likewise we need to validate the expected output for `notifier showconfig` comamnd

Reply:
@PrasadG193 OK will do that. but why do we need that switch case there?, its already in a for loop and the filters are same. we could just use the same method used in kubectl test na?

And I'm still figuring out how git and PR works. I shouldn't have deleted the branch gotta create a new PR now i guess